frontends/mit-open/webpack.config.js

```diff
@@ -39,6 +39,7 @@ const {
   EMBEDLY_KEY,
   CKEDITOR_UPLOAD_URL,
   SENTRY_DSN,
+  SENTRY_ENV,
```

Comment:
In addition to having the wrong var name in the github action, I neglected to actually pass it into the app.